tests/syntax/test_regions.py

```diff
@@ -167,6 +167,19 @@ def test_mesh_region_distribution():
     )
 
 
+def test_path_region_default_orientation():
+    ego = sampleEgoFrom(
+        """
+        import math
+        region = PathRegion(points=[(0,0,0), (1,1,math.sqrt(2)), (2,2,2*math.sqrt(2))])
+        ego = new Object in region
+    """
```

Comment:
Weird indentation here (should match opening quotes).

Reply:
It appears black insists on this style, as it won't even let me change it manually. Perhaps there's a setting we can tune?